### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +32,11 @@ contract ExampleContract {
     }
 
     function getBTCUSDPrice(bytes[] memory priceUpdateData) public returns (PythStructs.Price memory) {
-        // Update the prices to be set to the latest values. The `priceUpdateData` data should be
-        // retrieved from our off-chain Price Service API using the `pyth-evm-js` package.
+        // Update the prices to the latest available values and pay the required fee for it. The `priceUpdateData` data
+        // should be retrieved from our off-chain Price Service API using the `pyth-evm-js` package.
```
Comment (tompntn) [inline README.md:36]: Probably worth linking to the fee docs here https://docs.pyth.network/consume-data/on-demand#fees

File: README.md [hunk 2]
```diff
@@ -32,10 +32,11 @@ contract ExampleContract {
...
         // See section "How Pyth Works on EVM Chains" below for more information.
```
Comment (tompntn) [inline README.md:37]: Might be worth removing the on-demand explanation from this page, and replacing it with a link to https://docs.pyth.network/consume-data/on-demand so we don't have duplicate explanations

**Review by tompntn — APPROVED** [review]
Nice!

**Review by jayantk — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ali-bahjati) [reply README.md:36]: I think now that the link is below it makes sense. to keep it.

**Review by jayantk — APPROVED** [review]

**Pull request merged**